### DISCUSSION TITLE
Realme c21 Root.

### DISCUSSION THREAD

**Discussion by shihabahmmad** [community/community#83093] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

You need a lot of help. Cant root my phone realme c21 i need stock rom file RMX3201_11_C13

**Comment by Akash1134** [comment]
Hi @shihabahmmadsaikat ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
